lib/WiiExtension/WiiExtension.h

```diff
@@ -52,6 +58,10 @@
 #define WII_EXTENSION_DEBUG false
 #endif
 
+#ifndef WII_EXTENSION_ENCRYPTION
```

Comment:
If these don't do anything yet, might just put a comment above saying like

// NO IMPLEMENTATION YET

or something in case someone else picked up this code

Reply:
Something as food for thought is that I'm also going to be breaking this library out as standalone for RP2040 usage, so a lot of these would make more sense being included in a generic sense.  I'll make some commentary on it however.